tools/releaser/test/fixtures/activestorage/lib/active_storage/gem_version.rb

```diff
@@ -1,13 +1,13 @@
 # frozen_string_literal: true
 
-module Rails
+module ActiveStorage
   # Returns the currently loaded version of \Rails as a +Gem::Version+.
   def self.gem_version
     Gem::Version.new VERSION::STRING
   end
 
   module VERSION
-    MAJOR = 8
+    MAJOR = 1
```

Comment:
This doesn't seem right - only active storage is now 1.0.0.alpha?

Reply:
This is a fixture used for the tests.

Reply:
Ah, git got a little bit confused because I remove the root `version.rb` and it things I moved the file. `version.rb` was never used, so I removed it. Then, I added a fixture to test the `version_bump` method and it is the file that you see now.

Reply:
Ah ok, yea the git history made it look like an error. Everything else looks fine to me. 👍🏼